itest/test/spectre/spectre_test.py

```diff
@@ -641,18 +641,18 @@ def test_different_params_same_id(self):
         assert_is_in_spectre_cache("{}?ids={}&data=false".format(base_path, '6'))
 
     def test_with_invalid_id_when_cache_missing_ids_is_true(self):
-        base_path = '/bulk_requester_2'
+        path = '/bulk_requester_2/{ids}/v1?k1=v1'
 
         # 0 < ids < 1000 are valid
-        resp1 = get_through_spectre("{}/{}/v1?k1=v2".format(base_path, '10,5000,11'))
+        resp1 = get_through_spectre(path.format(ids='10,5000,11'))
```

Comment:
I know the variable name used to be resp1, but could we rename all the variables here that end in 1 or 2 to something better?